### DISCUSSION TITLE
Cloudflare: 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by dotfrankruan** [photoprism/photoprism#1704] [Q&A] [answered]
**What does not work as expected?**

I deployed photoprism using docker-compose on a ARM64 machine. Then I use NGINX to reverse proxy it. 
Cloudflare -> Host:443 -> Host:2342(photoprism)

**How can we reproduce it?**

Cannot. 

**What behavior do you expect?**

I can access it without ran into errors.

**Can you provide us with example files for testing or screenshots?**

My Compose file: https://pastebin.com/UNrcrFEE
Images: ![](https://ipfs-1.frank-ruan.com/ipfs/QmVRGpK35BEtHFyBVgXxD8YrYTLrSyH7cxaNoPbRTio7qr)

**What version you are using?**

I can't get into the software.

**Any other helpful information?**

If you are reporting a UI or performance issue, please always provide details about your...

- Web browsers: Google Chrome
- operating systems: Server: Ubuntu 20.04.3 LTS aarch64, Client: Ubuntu 20.04.4 LTS Desktop amd64 & KDE plasma
- installed memory: Server: 24 GB RAM, 100 GB Disk, Client: 8 GB RAM, 256 GB SSD & 500 GB HDD
- and processors: Server: Neoverse N1 (Run on Oracle Cloud A1 instance), Client: Intel Core i5-7200U

**Comment by graciousgrey** [comment]
Did you go through the checklists we provide -->  https://docs.photoprism.org/getting-started/troubleshooting/?

**Comment by godshades** [comment, marked as answer]
hi @orange2008 
i have same problem, did you find any solution

**Reply by godshades** [reply]
my quick fix is using public ip instead of local ip, lol

**Comment by Boxswurst** [comment]
Same Problem here. Plex for example is working fine.
